.env.sample

```diff
@@ -25,3 +25,10 @@ DB_HOST=db
 DB_PORT=5432
 # mysql | postgres | sqlite | mssql
 DB_DIALECT=postgres
+
+# Email settings
+EMAIL_API_KEY=
+EMAIL_API_URL=https://api.sendinblue.com/v3/smtp/email
+EMAIL_UPTIME_NOTIF_TEMPLATE_ID=2
+# Cron like sintax
+UPTIME_NOTIF_RECURRENCE=0 * * * *
```

Comment:
**This executes as shown on image below:**

![image](https://user-images.githubusercontent.com/23664028/74740077-7b74ae00-525a-11ea-9ef7-4f97414a591d.png)

Reply:
Thanks for the screenshot :D